### ISSUE TITLE
questions about label_frac.json and distant_data

### ISSUE THREAD

**Issue by WenxiongLiao** [rtmaww/EntLM#7]
How to build  label_frac.json and distant_data?

**Comment by rtmaww** [comment]
Hello. You can edit and run the first part of sh scripts/count_freq.sh to build label_frac.json. To build distant_data, you can refer to the BOND repo https://github.com/cliang1453/BOND .

**Comment by YiandLi** [comment]
Hello, I want to ask that:
 the distant dataset is directly **copy** the BOND dataset and **transfer tag_id to tag**, right?  Is there any filtering operation?

**Comment by rtmaww** [comment]
> Hello, I want to ask that: the distant dataset is directly **copy** the BOND dataset and **transfer tag_id to tag**, right? Is there any filtering operation?

Yes. We directly copied the BOND dataset without any filtering process.

**Comment by rzHub** [comment]
您好，在生成label_frac.json的时候，有一个参数是label_list_path，但dataset文件夹下没有这个参数指定的label_list.json文件，请问这个文件可以发一下吗？

**Comment by rtmaww** [comment]
> 您好，在生成label_frac.json的时候，有一个参数是label_list_path，但dataset文件夹下没有这个参数指定的label_list.json文件，请问这个文件可以发一下吗？

你好，该文件已经上传啦

**Comment by rzhub** [comment]
> > 您好，在生成label_frac.json的时候，有一个参数是label_list_path，但dataset文件夹下没有这个参数指定的label_list.json文件，请问这个文件可以发一下吗？
> 
> 你好，该文件已经上传啦

谢谢！

**Comment by HanbingWang2001** [comment]
您好，在运行bert_base时缺少label.txt这个文件，请问这个文件可以发一下吗？
